stackable-base/Dockerfile

```diff
@@ -16,11 +24,33 @@ ARG PRODUCT
 COPY stackable-base/stackable/dnf.conf /etc/dnf/dnf.conf
 
 RUN microdnf update && \
-    microdnf --assumeyes install shadow-utils && \
+    microdnf --assumeyes install \
```

Comment:
`tar` could be added because it is required for `kubectl cp`.

https://kubernetes.io/docs/reference/kubectl/generated/kubectl_cp/:
> Requires that the 'tar' binary is present in your container image. If 'tar' is not present, 'kubectl cp' will fail.